helpers/club.js

```diff
@@ -195,3 +195,22 @@ exports.getResources = async function (req, res) {
       embededlinks: getClub["embededlinks"],
     });
 };
+
+const getClubById = async (req, res) => {
+  // Get Id from the request
+  const { clubId } = req.body;
```

Comment:
when making GET requests, we accept queries from `req.query` instead of `req.body`. pulling params from the request body may work on postman, but it won't for packages like axios, which our frontend uses. i have fixed this for now, but it's something to keep in mind for the future!